app/src/main/java/org/fossasia/openevent/app/data/EventRepository.java

```diff
@@ -16,22 +20,24 @@
 import io.reactivex.Observable;
 import io.reactivex.android.schedulers.AndroidSchedulers;
 import io.reactivex.schedulers.Schedulers;
+import timber.log.Timber;
 
 public class EventRepository implements IEventRepository {
 
     public static final String ORGANIZER = "user";
-    public static final String EVENT = "event";
-    public static final String EVENTS = "events";
     public static final String ATTENDEES = "attendees";
 
+    private IDatabaseRepository databaseRepository;
```

Comment:
Can't we just use `inject` annotation instead of passing them directly to constructor?